pilot/pkg/features/pilot.go

```diff
@@ -279,6 +279,9 @@ var (
 	IstiodService = env.RegisterStringVar("ISTIOD_ADDR", "",
 		"Service name of istiod. If empty the istiod listener, certs will be disabled.")
 
+	IstiodServiceCustomHost = env.RegisterStringVar("ISTIOD_CUSTOM_HOST", "",
```

Comment:
Can you add a comment to explain what is the format of ISTIOD_CUSTOM_HOST and a few example values of ISTIOD_CUSTOM_HOST?

Reply:
added comment and examples

Reply:
'Additional host names added to Istiod certificate' ?